app/models/legacy/project_event.rb

```diff
@@ -53,7 +57,17 @@ def migration_step_data
     def migrate_step
       data = migration_step_data
       puts "ProjectStep[#{data[:id]}] #{data[:project_id]}"
-      ::ProjectStep.create(data)
+      step = ::ProjectStep.create(data)
+      step.parent = find_or_create_parent_group
```

Comment:
Am I correct in thinking that we are setting all step parents to the root group initially and then overwriting the parent_id later if an explicit one is given? I suppose that's fine but please add a comment as it could be confusing.